docs/canonicalk8s/charm/howto/ceph-csi.md

```diff
@@ -129,7 +129,107 @@ sudo k8s kubectl wait pod/pv-writer-test \
     --timeout 2m
 ```
 
+## Relate to multiple Ceph clusters
+
+So far, this guide demonstrates to how to integrate with a single ceph cluster
+represented by the single `ceph-mon` application. But {{product}} supports
+multiple ceph clusters via the same set of charms deployed again as different
+applications.
```

Comment:
Do you mean you are deploying more of the same charm just under a different name? Sorry I am a bit confused on this part. If you are I would suggest rewording it to something along the lines of: However, {{product}} supports multiple Ceph clusters. The same `ceph-mon` , `ceph-osd` and `ceph-csi` charms can be deployed again as different applications under different names.

Reply:
That is correct.  Same charms new names creates a new cluster.  I'll  use your suggestion to clarify this point